9.2-2.3/alpine/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM postgres:9.2-alpine
+MAINTAINER Régis Belson <me@regisbelson.fr>
+
+ENV POSTGIS_VERSION 2.3.1
+ENV POSTGIS_SHA256 0f1e0fe08f55b6db791d8600bb343e4f77590725acbbec99094a30b8b7439cd8
+
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories
```

Comment:
From https://github.com/docker-library/official-images#repeatability:
> For dependent packages installed by apt there is not usually a need to pin them to a version.
Since postgres is built from source in the base image, it's not going to be upgraded in any way by `apk upgrade`. I could also build the dependencies for postgis from source to use specific versions but I'm not sure it makes that much of a difference if the image provides the right versions of postgres and postgis anyway.